tests/test_api.py

```diff
@@ -126,10 +126,11 @@ def test_parent_links(self, client, entities, endpoint, parents):
         ('/sequencing-centers', ['sequencing_experiments', 'biospecimens']),
         ('/participants', ['diagnoses', 'phenotypes', 'outcomes',
                            'biospecimens']),
-        ('/biospecimens', ['diagnoses', 'biospecimen_genomic_files']),
+        ('/biospecimens', ['biospecimen_genomic_files']),
```

Comment:
Shouldn't the link to diagnoses exist, but not the biospecimen_genomic_files?

Reply:
ah, `diagnoses` link should exist and also `biospecimen_genomic_files`. i think you meant `biospecimen_diagnoses`?